### PR TITLE
Improve 'openio-admin meta2 rebuild'

### PR DESCRIPTION
**Pull request by AymericDu**
##### SUMMARY

`openio-admin meta2 rebuild` rebuilds a specified volume.

##### ISSUE TYPE

 - Feature Pull Request

##### COMPONENT NAME

- Python API
- Admin CLI

##### SDS VERSION

```
openio 5.0.0.0a2.dev3
```

### CODE REVIEW

File: bin/oio-meta2-rebuilder [hunk 1]
```diff
@@ -19,77 +19,86 @@
...
+
+    # local
+    parser.add_argument(
+        '--workers', type=int,
+        help='Number of workers. '
+             '(default=%d)' % Meta2Rebuilder.DEFAULT_WORKERS)
+    parser.add_argument(
+        '--references-per-second', type=int,
+        help='Max references per second. '
+             '(default=%d)' % Meta2Rebuilder.DEFAULT_ITEM_PER_SECOND)
```
Comment (fvennetier) [inline bin/oio-meta2-rebuilder]: We should talk about "items" now, I think.

File: bin/oio-meta2-rebuilder [hunk 2]
```diff
@@ -19,77 +19,86 @@
...
 
     return parser
 
 
 if __name__ == '__main__':
     args = make_arg_parser().parse_args()
 
+    if not any((args.volume, args.input_file)):
+        raise ValueError('Missing meta ID or input file')
```
Comment (fvennetier) [inline bin/oio-meta2-rebuilder]: Service ID or input file

File: oio/cli/admin/service_rebuild.py [hunk 3]
```diff
@@ -124,17 +124,41 @@ class Meta2Rebuild(ServiceRebuildCommand):
...
+
+        # input
+        parser.add_argument(
+            'service_id',
+            metavar='<service_id>',
+            help='ID of the service to rebuild')
+        # common
+        parser.add_argument(
+            '--rdir-fetch-limit', type=int,
+            help='Maximum of entries returned in each rdir response. '
```
Comment (fvennetier) [inline oio/cli/admin/service_rebuild.py]: Maximum number

and the default is not set

File: bin/oio-meta2-rebuilder [hunk 4]
```diff
@@ -19,77 +19,86 @@
...
-    log_parser.add_argument('--log-address',
-                            help="Log address")
-
-    descr = "Rebuild meta2 databases by setting 'last_rebuild' " \
-            "property in admin table, thus triggering a replication. " \
-            "And print the failed container IDs."
+    log_parser = make_logger_args_parser()
+    descr = """
+Rebuild meta2 databases by coppying the missing databases
+and synchronize this database.
```
Comment (fvennetier) [inline bin/oio-meta2-rebuilder]: Rebuild meta2 databases by replacing the missing replicas and triggering a synchronization.

File: oio/cli/admin/service_rebuild.py (hunk 3 above)
Comment (AymericDu) [reply oio/cli/admin/service_rebuild.py]: I chose to trust the default value chosen by the tool.
https://github.com/open-io/oio-sds/pull/1748/commits/0b23e4849bac7eefcfb69f507ea476e8ed6e3521#diff-c305d063edb11e4180028447625d07e4R42

**Pull request merged**